tools/make/docs.mk

```diff
@@ -0,0 +1,15 @@
+DOCS_DIR := docs
+DOCS_OUTPUT_DIR := docs/html
+
+.PHONY: docs
+docs: $(tools/sphinx-build)
+	tools/bin/sphinx-build -b html $(DOCS_DIR) $(DOCS_OUTPUT_DIR)
```

Comment:
i would use multiproc for rendering if you can - ie `-j auto` - not all extensions are compatible with that but i would start with multiproc and review if its a problem down the road

Reply:
👍